Formula/t/truetree.rb

```diff
@@ -0,0 +1,36 @@
+class Truetree < Formula
+  desc "Command-line tool for pstree-like output on macOS"
+  homepage "https://themittenmac.com/the-truetree-concept/"
+  url "https://github.com/themittenmac/TrueTree/archive/refs/tags/V0.8.tar.gz"
+  sha256 "38a6c3fa7328db16fdb66315ca97feb6cbbfb0a53d8af5aabd97e6d6a498a2c7"
+  license "CC-BY-NC-4.0"
```

Comment:
We actually can't accept this in Homebrew/core as this is not a DFSG-compatible license as required by https://docs.brew.sh/License-Guidelines

DFSG (and Fedora/GNU/FSF/...) apply a "free-as-in-speech" not "free-as-in-beer" interpretation which means all `CC-BY-NC*` licenses are not allowed since the NC refers to a non-commercial restriction.

For additional details, see:
* https://people.debian.org/~bap/dfsg-faq.html#no_commercial
* https://gitlab.com/fedora/legal/fedora-license-data/-/blob/main/data/CC-BY-NC-4.0.toml
* https://github.com/NixOS/nixpkgs/blob/master/lib/licenses.nix#L313-L316
* https://www.gnu.org/licenses/license-list.en.html#NonFreeDocumentationLicenses

---

If there is a signed binary, we can consider moving this Homebrew/Cask

Reply:
Got it, @jbradley89 what do you think?

Reply:
I did see the binary is signed so will try to migrate to Homebrew/Cask:
* https://github.com/Homebrew/homebrew-cask/pull/183422
* https://github.com/Homebrew/homebrew-core/pull/182464